DOCSUP-4435: Allow nullable key in MergeTree (#17345)
ClickHouse/ClickHouse: docs/en/operations/settings/settings.md

Default value: `\N`.

## allow_nullable_key {#allow-nullable-key}

> Enables or disables [Nullable](../../sql-reference/data-types/nullable.md#data_type-nullable) type as keys in [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engines-mergetree) tables.

Reviewer: ```suggestion
Allows using of the [Nullable](../../sql-reference/data-types/nullable.md#data_type-nullable)-typed values in a sorting and a primary key for  [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engines-mergetree) tables.
```

Possible values:

- 1 — `Nullable` type is allowed as keys.

> - 0 — `Nullable` type is not allowed as keys.

Reviewer: ```suggestion
- 0 — `Nullable`-type expressions are not allowed in keys.
```
